main-client.js

```diff
@@ -26,7 +27,20 @@ export const createMeteorNetworkInterface = (givenConfig) => {
   if (config.useMeteorAccounts) {
     networkInterface.use([{
       applyMiddleware(request, next) {
-        const currentUserToken = Accounts._storedLoginToken();
+        
+        // cookie login token created by meteorhacks:fast-render and caught during server-side rendering by rr:react-router-ssr
+        const { loginToken: cookieLoginToken } = config;
+        // Meteor accounts-base login token stored in local storage, only exists client-side
+        const localStorageLoginToken = Meteor.isClient && Accounts._storedLoginToken();
+        
+        // on initial load, prefer to use the token grabbed server-side if existing
+        let currentUserToken = cookieLoginToken || localStorageLoginToken;
+
+        // ...a login token has been passed to the config, however the "true" one is different ⚠️
+        if (Meteor.isClient && cookieLoginToken && cookieLoginToken !== localStorageLoginToken) {
```

Comment:
Hum... well, if I remove the verification on the apollo client config (and stop feeding it with a cookie client-side), after log-out the cookie used server-side on first load is still used client-side in my request.

When I log in back, it still use the previous token and not the one newly generated 🌮